packages/libro-lab/src/editor-viewer/code-editor-open-handler.ts

```diff
@@ -14,7 +14,7 @@ export class CodeEditorViewerOpenHandler extends NavigatableViewOpenHandler<Code
   id = CodeEditorViewerFactory;
 
   canHandle(uri: URI, options?: ViewOpenHandlerOptions) {
-    if (uri.scheme === 'file' && textFileTypes.includes(uri.path.ext)) {
+    if (uri.scheme === 'file' && textFileTypes.includes(uri.path.ext.toLowerCase())) {
```

Comment:
Using the `toLowerCase`  method for matching may have scenarios where it is not suitable. I noticed that there is language spec registration information later, which includes an extension list. Is it possible to merge these two? This way, extensions like .ts and .tsx can be compatible.